src/main/java/org/folio/rest/impl/TenantPermsAPI.java

```diff
@@ -718,7 +720,8 @@ public void postTenantpermissions(OkapiPermissionSet entity, Map<String, String>
                 return;
               }
```

Comment:
I have an issue with the preceding condition that checks for collision. Iv'e seen it reporting collision for mutable=false and also deprecated permission. For this reason (and as you did below),. I'd wisth we changed it to:
```java
         if (foundPerm.getModuleName() == null && !Boolean.TRUE.equals(foundPerm.getDeprecated())) {
             pgClient.rollbackTx(connection, rollback ->
            promise.fail("PermissionName collision with user-defined permission: " + perm.getPermissionName()));
         }
```

With this in mind the logic could be simplified.

Reply:
Can we make this change in scope of another bug/PR?

Reply:
Yes